windows/web/slimevr_web_installer.nsi

```diff
@@ -356,24 +286,20 @@ Section "SlimeVR Server" SEC_SERVER
     SetOutPath $INSTDIR
 
     DetailPrint "Downloading SlimeVR Server..."
-    NScurl::http GET "https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR.zip" "$TEMP\SlimeVR.zip" /CANCEL /RESUME /END
+    NScurl::http GET "https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR-win64.zip" "$TEMP\SlimeVR-win64.zip" /CANCEL /RESUME /END
     Pop $0 ; Status text ("OK" for success)
     ${If} $0 != "OK"
         Abort "Failed to download SlimeVR Server. Reason: $0."
     ${EndIf}
     DetailPrint "Downloaded!"
 
-    nsisunz::Unzip "$TEMP\SlimeVR.zip" "$TEMP\SlimeVR\"
+    nsisunz::Unzip "$TEMP\SlimeVR-win64.zip" "$TEMP\SlimeVR\"
     Pop $0
     DetailPrint "Unzipping finished with $0."
 
     DetailPrint "Copying SlimeVR Server to installation folder..."
     CopyFiles /SILENT "$TEMP\SlimeVR\SlimeVR\*" $INSTDIR
 
-    # Include modified run.bat that will run bundled JRE
-    File "run.bat"
-    File "run.ico"
```

Comment:
`run.ico` is still used for the shortcut and the registry entry for the program, it should still be included unless you change or remove those usages, I believe the shortcut should work fine without an icon as long as the exe has an icon?

Reply:
Fixed. Yes, exe should work.